drivers/sensor/paw3212/paw3212.c

```diff
@@ -0,0 +1,524 @@
+/*
+ * Copyright (c) 2019 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+ */
+
+#include <zephyr.h>
+#include <kernel.h>
+#include <sensor.h>
+#include <device.h>
+#include <spi.h>
+#include <gpio.h>
+#include <misc/byteorder.h>
+#include <sensor/pmw3360.h>
```

Comment:
do we need it here ?

Reply:
no, we would this sensor's header when we add attributes in next PR

Reply:
Fixed